docs/install_guides/installing-openvino-runtime.md

```diff
@@ -9,7 +9,6 @@
    Linux <openvino_docs_install_guides_installing_openvino_linux_header>
    Windows <openvino_docs_install_guides_installing_openvino_windows_header>
    macOS <openvino_docs_install_guides_installing_openvino_macos_header>
-   Raspbian OS <openvino_docs_install_guides_installing_openvino_raspbian>
 
```

Comment:
yeah, we need add Raspbian OS back.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/12190/commits/187fc1f52ad6c79e76fa7bb5d48387dc8b6879d6